### DISCUSSION TITLE
Hello, trying to install this mod and it says missing toml file, invalid mod

### DISCUSSION THREAD

**Discussion by maglore9900** [GriefPrevention/GriefPrevention#1256] [General]
Hello, trying to install this mod and it says missing toml file, invalid mod.

I am running 
minecraft 1.16.4
forge-1.16.4-35.1.37
and a few mods

am I missing a step in installation? Is there another mod that is needed to make this work?

I checked the issue tracker and the wiki and couldnt find an aswer.

thanks!

**Comment by bigpresh** [comment]
This is a Bukkit plugin for Bukkit/Spigot/Paper etc servers, not a Forge plugin/mod.

In general, you just drop the .jar file in your server's plugins dir, and then see https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration for how to configure it (although the default config should be enough to get you started!).

**Reply by maglore9900** [reply]
Ahh, Ill have to go research those. I didnt realize it had a different install method. I dont have a plugin folder as of now. I assume I need to take actions to support Bukkit plugins.

**Reply by maglore9900** [reply]
I got it now. Different type of server, with different sets of mods. This unfortunately wont work for me. Thank you for quickly answering.

**Reply by bigpresh** [reply]
You're welcome!
